### ISSUE TITLE
Trusted Community Members to Offer Insightful Trade Suggestions

### ISSUE THREAD

**Issue by kissofkill** [Revadike/SteamKey.Trade#61]
a board or group of people who are seniors / trusted members of community (based on the votes they have on their profile) who can comment on your trade if you press suggestion button on the trade page

**Comment by Revadike** [comment]
I don't think this is what we want. Users may do this on their own volition, as the trade chat is open and pulic for all. Besides, last thing we want is the policing we had in barter.vg.
